h/views.py

```diff
@@ -7,10 +7,13 @@
 from pyramid.events import ContextFound
 from pyramid.view import forbidden_view_config, notfound_view_config
 from pyramid.view import view_config
+import pyramid.response
 
 from . import session
 from .models import Annotation
 from .resources import Application, Stream
+import h.api_client.api_client as api_client
+import h.atom_feed
```

Comment:
I'd probably go one step short of using the full import path. Would rather see `from . import feed` here (as above with `from . import session`).